wait-for-it.sh

```diff
@@ -85,6 +87,10 @@ do
         WAITFORIT_QUIET=1
         shift 1
         ;;
+        -d | --delay)
```

Comment:
This code won't work if you provide `--delay=X`, where X is an integer. To align with how remaining arguments are parsed in this script, you need to split handling `-d` and `--delay` like this:

```sh
        -d)
        WAITFORIT_DELAY="$2"
        shift 2
        ;;
        --delay=*)
        WAITFORIT_DELAY="${1#*=}"
        shift 1
        ;;
```